spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -630,5 +630,52 @@ describe("kong.clustering.compat", function()
       assert.is_nil(assert(services[3]).ca_certificates)
     end)
 
-  end)
+  end)  -- describe
+
+
+  for _, flavor in ipairs({"traditional", "traditional_compatible"}) do
```

Comment:
Now the compat logic has no relationship with flavor, should we remove this for loop?